rdrf/rdrf/static/css/bootstrap-overrides.css

```diff
@@ -0,0 +1,72 @@
+/* Fix the admin menu that gets cut off to the right of the page since Bootstrap 5 no longer has .navbar-right styles. */
```

Comment:
I think these should eventually be moved into the main stylesheet / page-appropriate styles, perhaps as new classes.

It also seems counterproductive to fight bootstrap's defaults too much, so we might also want to examine if there's a more 'idiomatic' way to present the elements we've had to modify.